src/dune_lang/action.ml

```diff
@@ -631,4 +631,4 @@ let decode_pkg = make_decode decode_pkg
 let to_dyn a = to_dyn (encode a)
 let equal x y = Poly.equal x y
 let chdir dir t = Chdir (dir, t)
-let run prog args = Run (Slang.Literal prog :: List.map args ~f:(fun x -> Slang.Literal x))
+let run prog args = Run (Slang.Literal prog :: List.map args ~f:(fun x -> Slang.Literal x))
```

Comment:
Try to avoid including changes in PRs that don't relate to the thing that you're working on. One tip is to use `git add -p` which allows you to manually specify which changes you want to stage.